django_spanner/introspection.py

```diff
@@ -366,10 +365,15 @@ def get_key_columns(self, cursor, table_name):
             ON
                 rc.CONSTRAINT_NAME = ccu.CONSTRAINT_NAME
             WHERE
-                tc.TABLE_NAME="{table}"
-            """.format(
-                table=self.connection.ops.quote_name(table_name)
-            )
+                tc.TABLE_NAME=@table AND tc.TABLE_SCHEMA=@schema_name
+            """,
+            params={
+                "table": self.connection.ops.quote_name(table_name),
```

Comment:
here also: table name should not be quoted, and the lack of a test failure to me seems like a lack of test coverage

Reply:
Fixed as discussed